src/parkapi_sources/models/geojson_inputs.py

```diff
@@ -21,12 +21,23 @@
     UrlValidator,
 )
 
-from parkapi_sources.models import StaticParkingSiteInput
-from parkapi_sources.models.enums import ParkingSiteType
+from .enums import ParkingSiteType
+from .parking_site_inputs import StaticParkingSiteInput
 
 
 @validataclass
-class GeojsonFeaturePropertiesInput(ValidataclassMixin):
+class GeojsonBaseFeaturePropertiesInput(ValidataclassMixin):
+    def to_dict(self, *args, static_data_updated_at: datetime | None = None, **kwargs) -> dict:
+        result = super().to_dict()
```

Comment:
Suggestion: rename `result` to `feature_dictionary` or something like that, because that is more precise on what this variable is supposed to be. However the method is so small, that it's also ok for me if you leave it as it is.

Reply:
Same as above: it is the result, and result is shorter.